CMake/CMakeLibs.cmake

```diff
@@ -15,7 +15,7 @@ endmacro(SET_OSQUERY_COMPILE)
 
 macro(ADD_OSQUERY_PYTHON_TEST TEST_NAME SOURCE)
   add_test(NAME python_${TEST_NAME}
-    COMMAND python "${CMAKE_SOURCE_DIR}/tools/tests/${SOURCE}" --build "${CMAKE_BINARY_DIR}"
+    COMMAND python2 "${CMAKE_SOURCE_DIR}/tools/tests/${SOURCE}" --build "${CMAKE_BINARY_DIR}"
```

Comment:
This was just a quick hack to make it go.  It should really become $(PYTHON) so we can override it from the port.  Then, on the osquery side, if it isn't defined, default to "python"